webapp/package-lock.json

```diff
@@ -0,0 +1,18363 @@
+{
```

Comment:
Is this file necessary?

Reply:
No. I removed this and sent a new pull request with the changes